charts/operator/templates/_helpers.yaml

```diff
@@ -0,0 +1,129 @@
+{{- end -}}
+{{- end }}
+
+{{ define "manager.volumeMounts" -}}
+{{ printf "\n" -}}
+{{- if and .Values.airgapped .Values.runtimeVersions.configMapName -}}
+- mountPath: /etc/config/astro-runtime/metadata/versions.json
+  name: versions
+  subPath: versions.json
+{{- end }}
+{{ if and .Values.certManager.enabled (not .Values.webhooks.useCustomTlsCerts) -}}
+- mountPath: /tmp/k8s-webhook-server/serving-certs
+  name: cert
+  readOnly: true
+{{- else -}}
+- mountPath: /tmp/custom-certs
+  name: custom-certs
+  readOnly: true
+{{- end -}}
+{{- end }}
+
+{{ define "manager.args" -}}
+{{ printf "\n" -}}
+- --health-probe-bind-address=:8081
+- --leader-elect
+- --metrics-bind-address={{ if .Values.manager.metrics.useSecuredEndpoint }}127.0.0.1{{ else }}0.0.0.0{{ end }}:8080
+{{- if gt (len .Values.airflowNamespaces) 0 }}
+- --namespaces={{ join "," .Values.airflowNamespaces }}
+{{- end -}}
+{{- if .Values.airgapped }}
+- --airgapped
+{{- end -}}
+{{- if .Values.global.openshift }}
+- --openshift
+{{- end -}}
+{{- if .Values.kedaEnabled }}
+- --keda
+{{- end -}}
+{{- end }}
+
+{{ define "manager.containers" -}}
+- args: {{ include "manager.args" . | indent 2 }}
+  command:
+    - /manager
+  env:
+    - name: WEBHOOK_CERT_DIRECTORY
+      value: '{{- if .Values.webhooks.useCustomTlsCerts -}}/tmp/custom-certs{{- else -}}{{- end -}}'
+{{- range $i, $config := .Values.manager.env }}
+    - name: {{ $config.name }}
+      value: {{ $config.value | quote }}
+{{- end }}
+  image: '{{.Values.manager.image.repository}}:{{.Values.manager.image.tag}}'
+  livenessProbe:
+    httpGet:
+      path: /healthz
+      port: 8081
+    initialDelaySeconds: 15
+    periodSeconds: 20
+  name: manager
+  ports:
+    - containerPort: 9443
+      name: webhook-server
+      protocol: TCP
+  readinessProbe:
+    httpGet:
+      path: /readyz
+      port: 8081
+    initialDelaySeconds: 5
+    periodSeconds: 10
+  resources:
+    limits:
+      cpu: '{{ .Values.manager.resources.limits.cpu }}'
+      memory: '{{ .Values.manager.resources.limits.memory }}'
+    requests:
+      cpu: '{{ .Values.manager.resources.requests.cpu }}'
+      memory: '{{ .Values.manager.resources.requests.memory }}'
+  securityContext:
+    allowPrivilegeEscalation: false
+  volumeMounts: {{ include "manager.volumeMounts" . | indent 2 }}
+{{- if .Values.manager.metrics.useSecuredEndpoint }}
+- args:
+    - --secure-listen-address=0.0.0.0:8443
+    - --upstream=http://127.0.0.1:8080/
+    - --logtostderr=true
+    - --v=10
+  image: gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0
+  name: kube-rbac-proxy
+  ports:
+    - containerPort: 8443
+      name: https
+{{- end -}}
+{{- end }}
+
+{{- define "operator.serviceAccountName" -}}
+{{- if .Values.serviceAccount.create -}}
+    {{- default (printf "%s-airflow-operator-controller-manager" .Release.Name) .Values.serviceAccount.name }}
+{{- else -}}
+    {{- default "default" .Values.serviceAccount.name }}
+{{- end -}}
+{{- end -}}
```

Comment:
are we not linking with global.rbacEnabled

Reply:
done